ci-operator/jobs/openshift/cluster-operator/openshift-cluster-operator-tools-postsubmits.yaml

```diff
@@ -0,0 +1,34 @@
+postsubmits:
+  openshift/cluster-operator:
+  - agent: kubernetes
+    branches:
+    - tools
```

Comment:
@csrwng we need to get your repo migrated out to the `build_root.project_image` we built specifically for you and not have a separate config